### ISSUE TITLE
Blank Screen on Edit Project of Event

### ISSUE THREAD

**Issue by ddfridley** [DemocracyLab/CivicTechExchange#940]
When I try to [Edit] My project for Hacky New Year from this page https://www.democracylab.org/events/65/projects/281

I get to this page https://www.democracylab.org/events/65/projects/create/281

With an entirely blank viewport, and this error in the developer console:
```
vendors.bundle.js:1 @tryghost/content-api: The 'version' parameter has a deprecated format 'v{major}', please use 'v{major}.{minor}' format instead
ve @ vendors.bundle.js:1
vendors.bundle.js:1 TypeError: Cannot read properties of null (reading 'toString')
    at Function.value (main.bundle.js:formatted:25878:63)
    at t.i (main.bundle.js:formatted:25857:31)
    at t [as constructor] (vendors.bundle.js:1:829804)
    at new t (vendors.bundle.js:1:830722)
    at gi (vendors.bundle.js:1:756717)
    at Ra (vendors.bundle.js:1:774167)
    at bc (vendors.bundle.js:1:812531)
    at su (vendors.bundle.js:1:803778)
    at cu (vendors.bundle.js:1:803703)
    at Zc (vendors.bundle.js:1:800733)
    at vendors.bundle.js:1:752375
    at t.unstable_runWithPriority (vendors.bundle.js:1:827616)
    at Xo (vendors.bundle.js:1:752084)
    at Vo (vendors.bundle.js:1:752320)
    at Yo (vendors.bundle.js:1:752255)
    at Gc (vendors.bundle.js:1:797518)
ec @ vendors.bundle.js:1
bam.nr-data.net/jserrors/1/NRJS-1489159f1ea92734f50?a=337640472&v=1220.PROD&to=MVZTZhNTDxYDUxVdWwgcd0cPURUMDV5OV10QWlJGBFEJFRBfC1FXEkAfRAhXFhZYWQ9QUR4dVlcV&rst=10987&ck=0&s=f9c64988517f0056&ref=https://www.democracylab.org/events/65/projects/create/281:1          Failed to load resource: the server responded with a status of 403 (Forbidden)
```
Here is a screenshot of the code around line 25878
<img width="668" alt="image" src="https://user-images.githubusercontent.com/3317487/211940819-3e94530c-5040-40dc-bb1f-33b580961071.png">

One other piece of information is the the EnCiv project was the first to sign up for Hacky New Year, before some of the development changes. (So maybe something didn't get initialized in the database that would have for newer projects) I don't know if this is happening for other projects.

**Comment by PeterBreen** [comment]
@ddfridley did this get resolved for you? Specifically, were you able to edit your project successfully on or after Jan 11 @ 9:45pm PST when Marlon deployed an update?

i've done local testing and it looks fixed for me, but I don't want to close this out unless it works for you too :)

**Comment by ddfridley** [comment]
Works for me! Thanks. 



Get BlueMail for Android

**Comment by PeterBreen** [comment]
Great, thanks! Closed by #933.
